config.yaml

```diff
@@ -15,6 +15,7 @@ bgp:
   remote_as: 6789
   # override the peer IP to use instead of auto discovering
   peer_ip: 10.10.10.1
+  # at least 1 community must be specified
```

Comment:
thanks for making this explicit ! At some point in the near future , I should probably make it optional.

Reply:
Can more than one `peer_ip ` be defined?

Reply:
not as of today. Whats the use case ?

Reply:
Use case for multiple `peer_ips`: on a host with redundant networking, and each top-of-rack switch having a distinct IP, I need to peer with each top-of-rack switch to ensure the service is available if one switch fails.